ldes-server-domain/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/domain/ldes/retentionpolicy/RetentionPolicyConfig.java

```diff
@@ -28,7 +28,7 @@ public Map<String, List<RetentionPolicy>> retentionPolicyMap(
 				.stream()
 				.collect(Collectors.toMap(ViewSpecification::getName,
 						viewSpecification -> viewSpecification
-								.getRetentionPolicies()
+								.getRetentionConfigs()
 								.stream()
 								.map(this::getRetentionPolicy)
```

Comment:
Maybe it's because i'm missing the bigger picture, but are we mixing the term "retention policy" and "retention Config" now?

Reply:
I would like to make RetentionPolicy an interface so that we can have multiple of those: TimeBased, VersionBased, ...
RetentionConfig is part of the configuration and defines which policy needs to be used, for example TimeBased, and how it is configured, for example duration-in-seconds or period.